virtual_rainforest/models/plants/plants_model.py

```diff
@@ -93,9 +94,7 @@ class PlantsModel(BaseModel):
         "layer_fapar",
         "layer_leaf_mass",  # NOTE - placeholder resource for herbivory
         "layer_absorbed_irradiation",
-        "herbivory",
-        "transpiration",
-        "canopy_evaporation",
+        "evapotranspiration",
     )
```

Comment:
Did we not decided that `plants` only updates the transpiration?

Reply:
I see. This will need some adjustment from my side. The `evapotranspiration` is used in the hydrology model to estimate how much water is taken from the soil by the roots. So it should really only be transpiration, but at that time I thought that the plant model does calculate both but not return the components individually. If the canopy evaporation is calculated in the hydrology model, and the transpiration in the plants, we will only need the transpiration as input to the `setup_hydrology_input_current_timestep`. You can change it here if you want or I'll do it in a future PR

Reply:
I think we push this to a future PR 😄